README.md

```diff
@@ -4,53 +4,46 @@ A [Roc](https://github.com/roc-lang/roc) library for random number generation
 
 ## Status
 
-This works, but there's much more it could do.
+Forked from [JanCVanB/roc-random](https://github.com/JanCVanB/roc-random), updated to build package and documentation, and do some maintenance.
 
-Contributions & feedback are very welcome! :)
+This works, but there's much more it could do. Contributions & feedback are very welcome!
 
 ## Examples
 
-```coffee
-# Print a list of 10 random numbers in the range 25-75 inclusive.
-main =
-
-    # Initialise "randomness"
-    initialSeed = Random.seed16 42
-
-    # Create a generator for values from 25-75 (inclusive)
-    u16 = Random.u16 25 75
-
-    # Create a list of random numbers
-    result =
-        List.range { start: At 0, end: Before 10 }
-        |> List.walk { seed: initialSeed, numbers: [] } \state, _ ->
-
-            random = u16 state.seed
-            seed = random.state
-            numbers = List.append state.numbers random.value
+```roc
+import cli.Stdout
+import rand.Random
 
-            { seed, numbers }
-
-    # Format as a string
-    numbersListStr =
-        result.numbers
-        |> List.map Num.toStr
-        |> Str.joinWith ","
-
-    Stdout.line "Random numbers are: \(numbersListStr)"
+# Print a list of 10 random numbers in the range 25-75 inclusive.
+main! = \_args ->
+    random_numbers
+    |> List.map(Num.to_str)
+    |> Str.join_with("\n")
+    |> \numbers_list_str -> Stdout.line!("$(numbers_list_str)")
+
+numbers_generator : Random.Generator (List U32)
+numbers_generator =
+    Random.list(Random.bounded_u32(25, 75), 10)
+
+random_numbers : List U32
+random_numbers =
+    # we can ignore the updated seed value and just return the generated numbers
+    { value: numbers } = Random.step(Random.seed(1234), numbers_generator)
+
+    numbers
+
+expect
+    actual = random_numbers
+    actual == [52, 34, 26, 69, 34, 35, 51, 74, 70, 39]
 ```
 
 See the `examples/*.roc` files for more examples.
 
 ## Documentation
 
-See [the library documentation site](https://JanCVanB.github.io/roc-random)
+See [the library documentation site](https://lukewilliamboswell.github.io/roc-random/)
```

Comment:
Link is pointing to the wrong account.

Reply:
I've now fixed this via https://github.com/JanCVanB/roc-random/pull/26